pandas/tests/computation/test_eval.py

```diff
@@ -463,7 +463,7 @@ def check_compound_invert_op(self, lhs, cmp1, rhs):
 
             # make sure the other engines work the same as this one
             for engine in self.current_engines:
-                ev = pd.eval(ex, engine=self.engine, parser=self.parser)
+                ev = pd.eval(ex, engine=engine, parser=self.parser)
```

Comment:
I assume `engine` was supposed to be used here. This breaks some tests for me locally!

Reply:
i think you're probably right.  if its breaking tests, probably merits its own PR?

Reply:
I opened #40748 to keep track of this.